tmf/org.eclipse.tracecompass.tmf.core/src/org/eclipse/tracecompass/tmf/core/dataprovider/IDataProviderCapabilities.java

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Ericsson
+ *
+ * All rights reserved. This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0 which
+ * accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.tracecompass.tmf.core.dataprovider;
+
+import org.eclipse.tracecompass.tmf.core.config.ITmfDataProviderConfigurator;
+
+/**
+ * Interface to implement to indicate capabilities of a data provider, such as
+ * "canCreate" and "canDelete" capability.
+ *
+ * "canCreate" indicates that a given data provider can create a derived data
+ * provider. "canDelete" indicates that a the data provider can be deleted.
```

Comment:
"a the"

Reply:
Just to make sure, "canDelete" indicates that THIS data provider can be deleted, it is not related to deleting derived data providers that this data provider has created, correct? So it's the derived data providers that would have their own "canDelete" set to true?

Reply:
yes, the derived data provide would set canDelete to true and not the parent data provider.